apps/backoffice-v2/src/common/utils/get-severity-from-risk-score.ts

```diff
@@ -1,6 +1,12 @@
 import { Severity, TSeverity } from '@/common/types';
 
-export const getSeverityFromRiskScore = (riskScore: number): TSeverity | undefined => {
+export const getSeverityFromRiskScore = (
+  riskScore: number | null | undefined,
+): TSeverity | undefined => {
+  if (!riskScore && riskScore !== 0) {
+    return;
+  }
```

Comment:
Its nice just for getting this flow out of the way soon and explicitly but was this needed for some reason if <= 39 catches 0 and if no condition was hit the result is the same?

Reply:
For typing reasons, I added the `if (!risk)` line, but that would fail if the risk score is 0, so I added it as well to make sure we return the correct value.